src/windows/mega-macro.window.xml

```diff
@@ -12,6 +12,7 @@
 		<Layers>
 			<Layer level="ARTWORK">
 				<FontString name="$parentName" inherits="GameFontHighlightSmall" text="">
+					<Size x="36" y="36"/>
```

Comment:
Is 36 something that should be a variable/constant? I think there was something around for that iirc.

Reply:
Any idea where that'd be? I can't find any SetSize() calls and the other xml size tags are also hard coded. 36 seems to look pretty good at least.